cristin-import/src/test/resources/contributer_role_codes.txt

```diff
@@ -1,2 +1,3 @@
 REDAKTØR
-FORFATTER
\ No newline at end of file
+FORFATTER
```

Comment:
Denne txt fila er til glede for alle java programmere som har windows. Uten denne fila så vil den ikke kjøre. (hvis du har windows). Forrige utvikler som satt med repoet hadde windows. Denne fila kunne vært fjernet siden det er ingen med windows lengre, men samtidig hvorfor ikke bare la den stå der.